### ISSUE TITLE
Improvement on !diff function

### ISSUE THREAD

**Issue by hexedsilicon** [theneosloth/Bolas#34]
How to replicate:
!diff <url-1> <url-2> 
Diff too long

Here's a suggested improvement. Instead of `Diff too long` have this output in an embed.

```
  url 1 title          url 2 title
29 Creatures     30 Creatures     
31 Lands         30 Lands
18 Sorceries     20 Sorceries
22 Instants      20 Instants
Which would you like to see?
```
```
!diff creature

<list the creature differences>
```

**Comment by jeacaveo** [comment]
I like that idea, thing is we don't have any additional information on the cards besides the name.

@theneosloth do you think we can add a file with type and mana infor for each card (I can generate it from mtgjson and we'll have to keep it updated)? or can you think of an alternative?

I'm working on solving this, but so far all i have is splitting the diff to avoid the error.

**Comment by theneosloth** [comment]
Yeah I can't think of a solution that doesn't require a local database of some sort. We could try scraping the count from the deckbuilder websites but that would require a different approach for every website
